server/src/lib/model/user-client.ts

```diff
@@ -243,6 +245,18 @@ const UserClient = {
     return true;
   },
 
+  // update the challenge_team of the user who is already signed up but have not joined any team
+  // [BUG]: there are exceptions if the team doesn't exist
```

Comment:
We should add a constants file. I'll do that in a future diff, thanks for calling out the bug here. Please add:

```
// TODO(riley): Hook this up to a constants file.
```